packages/amplication-client/src/Resource/OverviewTile/OverviewTile.tsx

```diff
@@ -14,6 +14,18 @@ const AuthProviderLabels: { [k in EnumAuthProviderType]: string } = {
   [EnumAuthProviderType.Jwt]: "Passport JWT",
 };
 
+export enum EnumDbType {
+  mongo = "mongo",
+  postgres = "postgres",
+  mysql = "mysql",
+}
+
+const DbTypeLabels: { [k in EnumDbType]: string } = {
+  [EnumDbType.mongo]: "MongoDB",
+  [EnumDbType.mysql]: "MySQL",
+  [EnumDbType.postgres]: "PostgresSQL",
+};
```

Comment:
I would add it to the server in `packages/amplication-server/src/enums/EnumDbType`
```ts
export enum EnumDbType {
  MongoDB = 'MongoDB',
  MySQL = 'MySQL',
  PostgresSQL = 'PostgresSQL'
}

registerEnumType(EnumDbType, {
  name: "EnumDbType",
});
```

Then run `npx nx build amplication-server`

Reply:
by design the server should not be aware of the type of plugin used for the DB. I don't think we need to create this enum. 
I think it makes more sense to use fixed values in the client based on the installed plugin and in the future we will be able to add support for plugin categories to be able to identify the DB plugins more easily.


so my suggestion - get a list of installed plugins (use the hook usePlugin) and find one of the existing DB plugins and show the name of the one with the highest "order"